holidays/countries/ivory_coast.py

```diff
@@ -0,0 +1,128 @@
+#  License: MIT (see LICENSE file)
+from gettext import gettext as tr
+
+from holidays import FEB
+from holidays.groups import (
+    ChristianHolidays,
+    IslamicHolidays,
+    InternationalHolidays,
+    StaticHolidays,
+)
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON
+
+
+class IvoryCoast(
+    ObservedHolidayBase, ChristianHolidays, IslamicHolidays, InternationalHolidays, StaticHolidays
+):
+    """Ivory Coast holidays.
+
+    References:
+        * <https://natlex.ilo.org/dyn/natlex2/r/natlex/fe/details?p3_isn=44374>
+        * <https://www.droit-afrique.com/uploads/RCI-Decret-1996-205-jours-feries.pdf>
+        * <https://fr.wikipedia.org/wiki/F%C3%AAtes_et_jours_f%C3%A9ri%C3%A9s_en_C%C3%B4te_d%27Ivoire>
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Ivory_Coast>
+        * <https://www.timeanddate.com/holidays/ivory-coast/>
+        * <https://holidayapi.com/countries/ci/>
+    """
+
+    country = "CI"
+    # %s (estimated).
+    estimated_label = tr("%s (estimé)")
+    # %s (observed).
+    observed_label = tr("%s (observé)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observé, estimé)")
+    # Ivory Coast gained independence in 1960.
+    start_year = 1960
+    default_language = "fr"
+    supported_languages = ("en_CI", "en_US", "fr")
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        # the observed dates for the Ivory Coast's islamic holidays have been verified against
+        # local references (COSIM) and align with the default Umm al-Qura calculations.
+        IslamicHolidays.__init__(self, show_estimated=islamic_show_estimated)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, IvoryCoastStaticHolidays)
+        kwargs.setdefault("observed_rule", SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # Secular Holidays.
+
+        # New Year's Day.
+        self._add_new_years_day(tr("1er janvier"))
+
+        # Labor Day.
+        self._add_observed(self._add_labor_day(tr("Fête du travail")))
+
+        # Independence Day
+        self._add_observed(self._add_holiday_aug_7(tr("Fête Nationale")))
+
+        # Introduced in 1996. See details: https://en.wikipedia.org/wiki/C%C3%B4te_d%27Ivoire#History
+        if self._year >= 1996:
+            # National Peace Day.
+            self._add_holiday_nov_15(tr("Journée Nationale de la Paix"))
+
+        # Christian Holidays
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Lundi de Pâques"))
+        # Ascension Day.
+        self._add_ascension_thursday(tr("Jour de l’Ascension"))
+        # Pentecost Monday (Whit Monday).
```

Comment:
```suggestion
        # Whit Monday.
```
Standard name for this holidays in en_US locale.

Reply:
Yes, that's possible too. Depending on what we consider to be the source for `en_CI`. :)

Reply:
Should I keep or remove Pentecost Monday? :sweat_smile:

Reply:
For en_US, it's definitely Whit Monday, for en_CI - I'm also more likely to choose Whit Monday.